pkgs/development/web/newman/package.json

```diff
@@ -0,0 +1,3 @@
+[
+  { "newman": "git+https://github.com/postmanlabs/newman.git#v5.1.2" }
```

Comment:
What's wrong with just `"newman"`? It will save you the trouble of manually updating it yourself.

Reply:
I followed the spacegun package example in fact I'm ok to simplify it